lang/strings_russian.txt

```diff
@@ -0,0 +1,26 @@
+<?php
+
+# Copyright (c) 2011 Robert Munteanu (robert@lmn.ro)
```

Comment:
You probably want to put your name here, and update the copyright year.

Reply:
changed